sdk/core/azure-core-amqp/src/main/java/com/azure/core/amqp/implementation/AmqpChannelProcessor.java

```diff
@@ -450,8 +434,7 @@ public void onError(Throwable throwable) {
             if (!isCancelled()) {
                 processor.subscribers.remove(this);
                 actual.onError(throwable);
-                processor.logger.atInfo()
-                    .addKeyValue(SUBSCRIBER_ID_KEY, subscriberId)
+                processor.logger.atInfo().addKeyValue(SUBSCRIBER_ID_KEY, subscriberId)
```

Comment:
Should we make all chained calls (>=3) be on separate lines?

Reply:
Unfortunately, this only has a few configuration options


Always newline

```java
process
    .logger
    .atInfo()
    .addKeyValue(SUBSCRIBER_ID_KEY, subscriberId)
```


Split after first

```java
process.logger
    .atInfo()
    .addKeyValue(SUBSCRIBER_ID_KEY, subscriberId)
```


Current behavior / split only required

```java
 processor.logger.atInfo().addKeyValue(SUBSCRIBER_ID_KEY, subscriberId)
```

From looking at this before the first two make the code very long, which necessarily isn't a problem. But it's almost the inverse terseness issue where it's too spread out.